lib/web_console/templates/console.js.erb

```diff
@@ -864,11 +864,20 @@ REPLConsole.prototype.scrollToBottom = function() {
   this.outer.scrollTop = this.outer.scrollHeight;
 };
 
-// Change the binding of the console
+// Change the binding of the console.
 REPLConsole.prototype.switchBindingTo = function(frameId, callback) {
   var url = this.getSessionUrl('trace');
   var params = "frame_id=" + encodeURIComponent(frameId);
-  postRequest(url, params, callback);
+  var notification = document.createElement('div');
+  notification.className = "console-notification";
```

Comment:
Do we need to just feed a new input line?

Reply:
Currently the default unclassed console text color is black so it won't be visible. I can remove the div but the console text color will need to be changed